README.md

```diff
@@ -380,6 +378,8 @@ for before starting the migration process.
 
 After UCX assessment workflow is executed, the assessment dashboard will be populated with findings and common recommendations. See [this guide](docs/assessment.md) for more details.
 
+The UCX assessment workflow is intended to only be run once; re-running it is not supported. If the inventory and findings for a workspace need to be updated then UCX should first be [uninstalled](#uninstall-ucx) and reinstalled.
```

Comment:
```suggestion
The UCX assessment workflow is intended to only be run once; re-running it is not supported. If the inventory and findings for a workspace need to be updated, then UCX should be reinstalled by first [uninstalling](#uninstall-ucx) and then [installing](#install-ucx).
```

Reply:
@JCZuurmond: You're right, it was a little clumsy. Using the suggestion as inspiration I've rephrased it slightly and removed the passive voice. Does it seem better now?